cluster-api/src/main/java/io/aiven/klaw/clusterapi/services/SchemaService.java

```diff
@@ -336,4 +338,33 @@ protected ClusterStatus getSchemaRegistryStatus(
       return ClusterStatus.OFFLINE;
     }
   }
+
+  public ApiResponse deleteSchema(ClusterTopicRequest clusterTopicRequest) {
+    String suffixUrl =
+        clusterTopicRequest.getSchemaEnv()
+            + "/subjects/"
+            + clusterTopicRequest.getTopicName()
+            + "-value";
+    Pair<String, RestTemplate> reqDetails =
```

Comment:
I think changing "subjects" and "-value" into constants is good practice here.

Reply:
Agree, done.